aggregator/src/app/BundleTable.ts

```diff
@@ -82,6 +103,7 @@ function toInsertRawRow(row: InsertRow): InsertRawRow {
     bundle: JSON.stringify(bundleToDto(row.bundle)),
     eligibleAfter: toUint256Hex(row.eligibleAfter),
     nextEligibilityDelay: toUint256Hex(row.nextEligibilityDelay),
+    receipt: JSON.stringify(row.receipt),
```

Comment:
Assume there is reason for this. Why do we only need to add the receipt property here and not the status as well? Same question for `toRawRow` below

Reply:
Yeah we don't have to have the status here because I'm not transforming the status value like I am the receipt value.  Since the receipt is an object I'm stringifying it to save it in the DB.  Then we have to parse that string when we get it back from the DB into an object.  Since the status is already a string it gets added to the object with the spread of the row object (`...row,`).

Saving the receipt as a string is kind of a 'quick win'. rather than having to do something like create a new `receipt` table where every possible value on the receipt is a column.  We do the same thing with the bundle column in the bundles table.